genesyscloud/architect_flow/resource_genesyscloud_flow.go

```diff
@@ -0,0 +1,207 @@
+package architect_flow
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	"terraform-provider-genesyscloud/genesyscloud/util/files"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v121/platformclientv2"
+)
+
+func getAllFlows(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
+	resources := make(resourceExporter.ResourceIDMetaMap)
+	p := getArchitectFlowProxy(clientConfig)
+
+	flows, err := p.GetAllFlows(ctx)
+	if err != nil {
+		return nil, diag.Errorf("failed to get architect flows %v", err)
+	}
+
+	for _, flow := range *flows {
+
+		//DEVTOOLING-393:  Putting this in here to deal with the situation where Cesar's BCP app is reliant on the naming structure
+		//This should be removed once the CX as Code architect export process is complete and will export files with the type in the name.
+		overrideBCPNaming := os.Getenv("OVERRIDE_BCP_NAMING")
```

Comment:
This is the code I wrote to deal with DEVTOOLING-393.  Basically I look to see if the OVERRIDE_BCP_NAMING env has been set and if it has, then deal we use the old naming standard